src/FakeStateMachine.ts

```diff
@@ -1,10 +1,11 @@
 import * as jsonpath from 'jsonpath';
 import { RunStateResult } from './RunStateResult';
 
-interface Definition {
+export interface Definition {
```

Comment:
[ask] Is this `export` necessary?

Reply:
It is nice to have since developer can define some functions to prepare state machine definition, and can use this interface as return type.

Reply:
That makes sense